addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -607,11 +607,14 @@ export class AnalyticDistribution extends Component {
 
     onWindowClick(ev) {
         /*
-        Dropdown should be closed only if all these condition are true:
+        Dropdown should be closed only if all these conditions are true:
             - dropdown is open
             - click is outside widget element (widgetRef)
-            - there is no active modal containing a list/kanban view (search more modal)
-            - there is no popover (click is not in search modal's search bar menu)
+            - Either:
+                - The click is not inside an active modal without a list/kanban view (search more modal)
+                    and not inside a popover (search bar menu)
```

Comment:
Here, shouldn't be "with a list/kanban view"? Because the search more modal has a list view, so a click there does not close the dropdown :thinking: 

```suggestion
                - The click is not inside an active modal with a list/kanban view (search more modal)
                  and not inside a popover (search bar menu)
```

Reply:
yes, that is correct :sweat_smile: